php/src/app/handler/Post/GetPostListHandler.php

```diff
@@ -0,0 +1,25 @@
+<?php 
+namespace App\Handler\Post;
+require_once __DIR__ . '/../../vendor/autoload.php';
+
+use App\Repo;
+
+class GetPostListHandler{
+    private \PDO $pdo;
+
+    function __construct()
+    {
+        //pdoの初期化
+        try{
+//            $this->pdo = init_pdo();
+        }catch (\Exception $e){
+            throw $e;
+        }
+    }
+
+    public function run(){
+//        $posts = Repo\Posts::get_posts($this->pdo);
+        echo '<p>hello world </p>';
```

Comment:
Handlerの責務は、リクエストを受け取ってレスポンスを返すことです。echoをすると「レスポンスを出力する」という責務まで行うことになります。
一旦はstringで良いので、値を返す関数にしましょうか。

```suggestion
        return '<p>hello world </p>';
```